cmd/memparse.go

```diff
@@ -49,6 +49,22 @@ func MemParse() *cobra.Command {
 		"Specify the output file to be written to",
 	)
 
+	flags.StringVarP(
+		searchPattern,
+		"search",
+		"s",
+		"",
+		"Search for a pattern in memory pages (accepts plain string or regex)",
```

Comment:
```suggestion
		"Search for a regex pattern in memory pages",
```
A plain string is also valid regex ;)